draft-ietf-quic-recovery.md

```diff
@@ -401,20 +401,34 @@ decrypt because 1-RTT packet protection keys are not yet available to it. In
 such cases, an endpoint SHOULD subtract such local delays from its RTT sample
 until the handshake is confirmed.
 
-smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.
+Similar to {{?RFC6298}}, smoothed_rtt and rttvar are computed as follows.
 
-When there are no samples for a network path, and on the first RTT sample for
-the network path:
+An endpoint needs to initialize the RTT estimator during connection
+establishment and when the estimator is reset during connection migration
+(Section 9.4 of {{QUIC-TRANSPORT}}). Before any RTT samples are available for a
+new path or when the estimator is reset, the estimator is initialized using an
+initial RTT value of 333ms; see {{pto-handshake}}.
```

Comment:
This is slightly different from what we say in handshakes and new paths(pto-handshake).  I'd suggest referencing that rather than re-specifying 333ms here.
```suggestion
(Section 9.4 of {{QUIC-TRANSPORT}}). Before any RTT samples are available for a
new path or when the estimator is reset, the estimator is initialized using the
initial RTT; see {{pto-handshake}}.
```

FYI, existing text is:
   Resumed connections over the same network MAY use the previous
   connection's final smoothed RTT value as the resumed connection's
   initial RTT.  When no previous RTT is available, the initial RTT
   SHOULD be set to 333ms, resulting in a 1 second initial timeout, as
   recommended in [RFC6298].

Reply:
Fair enough.